### DISCUSSION TITLE
paperclip & okapi

### DISCUSSION THREAD

**Discussion by somehowchris** [paperclip-rs/paperclip#372] [Ideas]
I come from the world of rocket and used to use [okapi](https://github.com/GREsau/okapi) which is easy to timplement, scalable and usable.

Now I stumbled across paperclip and like a lot of things specially on the client side, is there any way to have paperclip catch up support with okapi? Would love to help but my current overview of paperclip is slim

**Comment by tiagolobocastro** [comment]
Hey @somehowchris, welcome to paperclip!

To be honest I haven't delved into the client side of paperclip, would you be able to highlight what paperclip is lacking compared to okapi?

**Reply by somehowchris** [reply]
Okapi is just the server side equivilant of paperclip, it would be awesome they were up at the same level, not that one needs to actually copy a feature from the other. Okapi currently has no client side solution

**Reply by tiagolobocastro** [reply]
Ah ok thought you were talking about the client side. 
I've never used okapi, what is it that paperclip needs to catch up support with?

**Reply by somehowchris** [reply]
The big thing is [rocket](https://rocket.rs) support

**Reply by tiagolobocastro** [reply]
I suspect that would be quite a large lump of work that would need to be done as we currently only support actix-web. I don't know if someone would be willing to undertake this since okapi already exists for rocket.
